docs/src/content/docs/resources/glossary.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Glossary
+---
+
+import { LinkOut } from '@interledger/docs-design-system'
+
+## Account servicing entity (ASE)
+
+An account servicing entity provides and maintains a payment account for a payer and payee, and is a regulated entity in the country/countries it operates.
+
+## Authorization server (AS)
+
+An authorization server issues tokens to clients, which can then be used to perform authorized actions on resource servers. In the context of Open Payments, the authorization server grants permission for a client to access the Open Payments APIs and the `incoming-payment`, `quote`, and `outgoing-payment` resources. Open Payments leverages GNAP as the mechanism for delegating authorization.
+
+## Client
+
+A client is an application or service that interacts with the authorization server to obtain grants and tokens. A client uses these tokens to access resources or perform transactions on behalf of a user or system.
+
+## Grant Negotiation and Authorization Protocol (GNAP)
+
+The Grant Negotiation Authorization Protocol (GNAP) defines a mechanism for delegating authorization to a piece of software, and conveying the results and artifacts of that delegation to the software. This delegation can include access to a set of APIs as well as subject information passed directly to the software. For more information, see the [specification](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol-12).
+
+## Incoming payment resource
+
+An incoming payment resource is an object that represents a payment being received by an entity. This resource contains information about the incoming payment, such as the amount, currency, receiver's wallet address, and payment status. It is used to track and manage payments that are expected to or have been received.
+
+## Interactive grant
```

Comment:
Building off Mohammed's comment about whether we should include an entry for this in the glossary...

I think a good compromise may be to remove the entry, and in the outgoing payment resource entry, link the word "consent" to /introduction/grants/#identity-providers. 

Within that section, there's a tooltip that defines an interactive grant as requiring explicit interaction/consent from the resource owner before an access token can be issued. This way the reader can see the grant along with the thing that manages consent -- the identity provider.

Reply:
I like that approach. Removing entry for "interactive grant" and linking to identity providers in the "outgoing payment resource" entry.